File: 백준/Bronze/14470. 전자레인지/전자레인지.c
#include <stdio.h>

int main(void) { 
   int a, b, c, d, e;
   scanf("%d", &a);
   scanf("%d", &b);
   scanf("%d", &c);
   scanf("%d", &d);
   scanf("%d", &e);

   int sum = 0;
   if(a < 0){
      int x = 0 - a;
      sum += x * c;
      a = 0;
   }

   if(a == 0){
      sum += d;
   }

   sum += (b - a) * e;

   printf("%d", sum);

   return 0;
}

File: 백준/Bronze/14470. 전자레인지/전자레인지.py
a = int(input())
b = int(input())
c = int(input())
d = int(input())
e = int(input())

sum = 0

if a < 0:
  sum += (0 - a) * c
  a = 0

if a == 0:
  sum += d

sum += (b - a) * e

print(sum)